site/docs/reference/Connectors/materialization-connectors/amazon-s3-csv.md

```diff
@@ -0,0 +1,96 @@
+| **`/awsSecretAccessKey`** | AWS Secret Access key | Secret Access Key for writing data to the bucket.                                                                                             | string  | Required         |
+| **`/region`**             | Region                | Region of the bucket to write to.                                                                                                             | string  | Required         |
+| **`/uploadInterval`**     | Upload Interval       | Frequency at which files will be uploaded.                                                                                                    | string  | 5m               |
+| `/prefix`                 | Prefix                | Optional prefix that will be used to store objects.                                                                                           | string  |                  |
+| `/fileSizeLimit`          | File Size Limit       | Approximate maximum size of materialized files in bytes. Defaults to 10737418240 (10 GiB) if blank.                                           | integer |                  |
+| `/endpoint`               | Custom S3 Endpoint    | The S3 endpoint URI to connect to. Use if you're materializing to a compatible API that isn't provided by AWS. Should normally be left blank. | string  |                  |
+| `/csvConfig/delimiter`    | Delimiter             | Character to separate columns within a row. Defaults to a comma if blank. Must be a single character with a byte length of 1.                 | integer |                  |
+| `/csvConfig/nullString`   | Null String           | String to use to represent NULL values. Defaults to an empty string if blank.                                                                 | integer |                  |
+| `/csvConfig/skipHeaders`  | Skip Headers          | Do not write headers to files.                                                                                                                | integer |                  |
+
+#### Bindings
+
+| Property    | Title | Description                                    | Type   | Required/Default |
+|-------------|-------|------------------------------------------------|--------|------------------|
+| **`/path`** | Path  | The path that objects will be materialized to. | string | Required         |
+
+### Sample
+
+```yaml
+materializations:
+  ${PREFIX}/${mat_name}:
+    endpoint:
+      connector:
+        image: "ghcr.io/estuary/materialize-s3-csv:dev"
+        config:
+          bucket: bucket
+          awsAccessKeyId: <access_key_id>
+          awsSecretAccessKey: <secret_access_key>
+          region: us-east-2
+          uploadInterval: 5m
+    bindings:
+      - resource:
+          path: ${COLLECTION_NAME}
+        source: ${PREFIX}/${COLLECTION_NAME}
+```
+
+## File Names
```

Comment:
I'd clarify in this example that all those csv segments are from the same Collection, so folks don't expect a 1:1 relationship between collection and csv file. Maybe this could be made clearer in the beginning section.

Or maybe a light visualization like

```
┌───────────────────────────────┐
│         Collection            │
│     (single data source)      │
└──────────────┬────────────────┘
               │
               │
               ▼
    ┌───────────────────────────┐
    │ Materialized Files        │
    │ (multiple CSV segments)   │
    └───────────────────────────┘
               │
               │
               ▼
    ┌──────────────────────────────────────────┐
    │ bucket/prefix/path/v0000000000/          │
    │      00000000000000000000.csv            │
    ├──────────────────────────────────────────┤
    │      00000000000000000001.csv            │
    ├──────────────────────────────────────────┤
    │      00000000000000000002.csv            │
    └──────────────────────────────────────────┘

```

Reply:
I added some clarification here and to the beginning section that will hopefully help make it more clear that each collection will produce many files.